### PR TITLE
Update gusb.rb

### PR DESCRIPTION
**Pull request by lovedheart**
Fix dependency

### CODE REVIEW

**Comment by uberhacker** [comment]
Does this fix #1802?

File: packages/gusb.rb [hunk 1]
```diff
@@ -3,16 +3,13 @@
 class Gusb < Package
   description 'GUsb is a GObject wrapper for libusb1'
   homepage 'https://www.openhub.net/p/gusb'
-  version '0.2.11'
+  version '0.2.11-0'
   source_url 'https://github.com/hughsie/libgusb/archive/gusb_0_2_11.tar.gz'
```
Comment (uberhacker) [inline packages/gusb.rb]: Update available?  https://github.com/hughsie/libgusb/archive/0.3.0.tar.gz

Comment (lovedheart) [reply packages/gusb.rb]: @uberhacker  No. I won't update it. I have tried that and get the errors, which I can not fix right now.

Comment (uberhacker) [reply packages/gusb.rb]: No worries.  It looks more like a beta release anyway.

**Comment by uberhacker** [comment]
@lovedheart: Did you plan on adding binaries to this PR or do you want me to merge it and provide later?

**Comment by lovedheart** [comment]
merge it right now

**Pull request merged**